examples/remote_control_cozmo.py

```diff
@@ -679,6 +695,15 @@ def handle_setMouseLookEnabled():
     return ""
 
 
+@flask_app.route('/setHeadLightEnabled', methods=['POST'])
+def handle_setHeadLightEnabled():
+    '''Called from Javascript whenever head-light enabled is toggled'''
```

Comment:
"head-light" should be "headlight"

I think this is not quite precise: "Called from Javascript whenever head-light enabled is toggled". Maybe instead: "Called from Javascript when headlight is toggled" or "when headlight mode is toggled"?